mon-pix/translations/en.json

```
@@ -218,6 +219,22 @@
   "navigation": {
     "homepage": "Home page",
     "back-to-profile": "Back to profile",
-    "pix": "Pix"
+    "main": {
+      "code": "I have a code",
+      "help": "Help",
+      "link-help": "https://pix.fr/aide",
+      "profile": "Profile",
+      "start-certification": "Certification"
+    },
+    "not-logged": {
+      "login": "Login",
+      "signup": "Sign up"
+    },
+    "pix": "Pix",
+    "user": {
+      "certifications": "Mes certifications",
+      "logout": "Se déconnecter",
+      "tutorial": "Mes tutos"
```

```suggestion
      "tutorials": "Mes tutos"
```
pour être cohérent avec `certifications`